setup_cfg_fmt.py

```diff
@@ -372,6 +373,11 @@ def format_file(
     if setup_requires:
         cfg['options']['setup_requires'] = '\n'.join(setup_requires)
 
+    if cfg.has_section('options.extras_require'):
+        for key in cfg['options.extras_require'].keys():
```

Comment:
does this work without the call to `.keys()` -- usually calling keys is unnecessary

Reply:
Indeed it does, removed it.